webapp/src/pages/welcome/welcomePage.tsx

```diff
@@ -35,25 +36,30 @@ const WelcomePage = React.memo(() => {
     return (
         <div className='WelcomePage'>
             <div>
-                <h1 className='text-heading6'>
+                <h1 className='text-heading9'>
                     <FormattedMessage
                         id='WelcomePage.Heading'
                         defaultMessage='Welcome To Boards'
                     />
                 </h1>
-                <div className='text-base'>
+                <div className='WelcomePage__subtitle'>
                     <FormattedMessage
                         id='WelcomePage.Description'
                         defaultMessage='Boards is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view'
                     />
                 </div>
 
-                <div className='WelcomePage__image'>
-                    <img
-                        src={BoardWelcomePNG}
-                        alt='Boards Welcome Image'
-                    />
-                </div>
+                <img
+                    src={BoardWelcomePNG}
+                    className='WelcomePage__image WelcomePage__image--large'
+                    alt='Boards Welcome Image'
+                />
+
+                <img
+                    src={BoardWelcomeSmallPNG}
+                    className='WelcomePage__image WelcomePage__image--small'
+                    alt='Boards Welcome Image'
+                />
```

Comment:
Hey Asaad, 

I'm okay with the work done however I think we are unnecessarily rendering 2 components and hiding one when we could render one or the other conditionally based on Screen size. 

Do you think we can do something like here https://blog.logrocket.com/developing-responsive-layouts-with-react-hooks/ to render one or the other? That way it isn't confusing when inspecting the elements and noticing both are rendered but one is simply hidden.

Thoughts?

Reply:
@hahmadia Not sure I want to add a resize handler for something so simple. We would be running javascript code on resize which I'm not sure is necessary.

But I'm fine if you guys think adding the resize handler and reloading the images as the user resizes the screen is a better option for this.

Reply:
Nah I agree I think you're right. Rethinking it, it's pretty simple and adding more code could possibly complicate it more than it needs too. Can you just add a comment for each component to specify when it will render?